docs/pages/server-access/guides/openssh.mdx

```diff
@@ -340,6 +340,89 @@ authenticate the host via the certificate we generated earlier.
 
 </Details>
 
+<Details title="Using Trusted Clusters?">
+
+The configuration file that `tsh config` generates uses a `ProxyCommand`
+directive to connect to SSH servers in a leaf cluster through the root cluster's
+Proxy Service.
+
+In scenarios where Proxy Service instances in a leaf cluster are reachable by
+SSH clients, you might prefer to connect directly through the leaf proxies to
+achieve lower latency.
+
+To support this scenario, the `tsh proxy ssh` command provides the `-J` flag.
+This flag points to the public address of a cluster's Proxy Service and indicates 
+that the OpenSSH client will be connecting through that proxy instead of the proxy
+you're currently logged into:
+
+```txt
+Host *.{{ .NodeName }}.{{ .ClusterName }}
+   Port 3022
+   ProxyCommand tsh proxy ssh -J proxy.leaf1.example.com:443 %r@%h:%p
+```
+
+The `-J` flag dynamically determines the jump proxy address and the name
+of the host to connect to using the SSH config's full hostname spec (`%h:%p`)
+and the `{{proxy}}` template variable:
+
+The jump proxy address and the node name to connect to can be dynamically
+determined from the SSH config's full hostname spec (`%h:%p`) using `{{proxy}}`
+template variable:
+
+```txt
+Host *.example.com
+   Port 3022
+   ProxyCommand tsh proxy ssh -J {{proxy}} %r@%h:%p
+```
+
+You can define the rules of how to parse the proxy address and node name
+from the full host in the `tsh` configuration file (`~/.tsh/config/config.yaml`
```

Comment:
Is "full host" the same as "hostname provided in an `ssh` command"?

Reply:
No, it's referring to `%h:%p`, and the ssh `Hostname` is just the `%h`. I've changed the wording to "host server address" for clarity.